cardigan/config/decorators.tsx

```diff
@@ -18,53 +18,65 @@ export const ContextDecorator: FunctionComponent<Props> = ({ children }) => {
   );
 };
 
+const ReadMeInfo = ({ Readme }: { Readme: ComponentType }) => {
+  return (
+    <Space
+      v={{
+        size: 'xl',
+        properties: ['margin-top'],
+      }}
+      h={{ size: 'm', properties: ['padding-left', 'padding-right'] }}
+      className="body-text"
+      style={{
+        border: '1px solid #eee',
+        borderRadius: '6px',
+        transform: 'scale(0.8)',
+        transformOrigin: 'top left',
+        position: 'relative',
+      }}
+    >
+      <span
+        id="readme"
+        style={{
+          position: 'absolute',
+          fontSize: '12px',
+          fontWeight: 'bold',
+          color: '#aaa',
+          top: '-10px',
+          padding: '0 6px',
+          background: '#fff',
+        }}
+      >
+        README
+      </span>
+      <Space v={{ size: 'm', properties: ['padding-top'] }}>
+        <Readme />
+      </Space>
+    </Space>
+  );
+};
+
 type ReadmeDecoratorProps = {
   WrappedComponent: ComponentType;
   args: Record<string, unknown>;
   Readme: ComponentType;
+  isReadMeBefore?: boolean;
```

Comment:
> This would be much easier to review if it was split into multiple PRs for different changes

I've done so in the commits, which I should've mentioned.

Reply:
Ah, I should have paid more attention to the commit history.

Reply:
I just thought the cleanup would be simpler and as I went it kept seeing more things, but I'll try to restrain the one PR thing moving forward, it is a bit intense 😄